src/query/execute_statement.c

```diff
@@ -18135,7 +18168,7 @@ get_dblink_info_from_dbserver (PARSER_CONTEXT * parser, const char *server, DB_V
 	  sprintf (dblink_url, dblink_url_fmt_none, host, port_no, dbname);
 	}
 
-      error = db_make_string (&(out_val[0]), dblink_url);
+      error = db_make_string_copy (&(out_val[0]), dblink_url);
```

Comment:
This is a bug fix, not a feature change.
There was a problem that it was not guaranteed when exiting the function when using a local array.